boards/nxp/imx93_evk/imx93_evk_mimx9352_m33_ddr_defconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CLOCK_CONTROL=y
+CONFIG_PINCTRL=y
+CONFIG_SERIAL=y
```

Comment:
Remove CONFIG_PINCTRL from boards config. 

See 82a192c8a92b66a2b commit for example.